### PR TITLE
Add Userland fios2.h

### PR DESCRIPTION
**Pull request by KuromeSan**

### CODE REVIEW

File: include/psp2/fios2.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  // If both src and dst have the same modification time, dst is used.
+  // If no file exists at src or dst, dst is used; if no file exists at dst, but a file exists at src, src is used. Writes go to dst.
+  SCE_FIOS_OVERLAY_TYPE_NEWER       = 2,
+
+  // src merges with dst. Reads check src first, then dst. Writes go to src.
+  SCE_FIOS_OVERLAY_TYPE_WRITABLE    = 3
+} SceFiosOverlayType;
+  
+typedef struct SceFiosOverlay {
+  uint8_t type; // see SceFiosOverlayType
```
Comment (joel16) [inline include/psp2/fios2.h:38]: Shouldn't these be using SceUint* prefix?

Comment (KuromeSan) [reply include/psp2/fios2.h]: it wasn't used in the kernel header, but ill add it anyway.

**Comment by KuromeSan** [comment]
oh the library is SceFios2User not SceFios2,

**Comment by KuromeSan** [comment]
psp2/fios2.h: Unknown group SceFios2User
ok i have no idea what causes this, clearly

**Comment by d3m3vilurr** [comment]
@KuromeSan you should add the group definition in `docs/definitions.dox`

**Comment by KuromeSan** [comment]
whats the status on this?

File: docs/definitions.dox [hunk 2]
```diff
@@ -135,7 +135,10 @@
  *
  *     \defgroup SceFios2Kernel File System Overlay Library
  *      Add/Remove file system overlay
-
+ 
+        \defgroup SceFios2User File System Overlay Library
+ *      Add/Remove file system overlay
+
```
Comment (d3m3vilurr) [inline docs/definitions.dox:141]:
```suggestion
 *
 *      \defgroup SceFios2User File System Overlay Library
 *      Add/Remove file system overlay
 *
```
@Princess-of-Sleeping shared it.
you need to add `*` into each lines.

**Comment by d3m3vilurr** [comment]
replaced #751

**Pull request closed without merging**